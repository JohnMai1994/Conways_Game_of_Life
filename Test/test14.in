new 10
init 
8 4 3 8 3 4 2 3 4 5 3 2 9 4
9 0 8 7 2 9 3 0 4 1  3 4 7 5
-1 -1
print
steps 5
print
