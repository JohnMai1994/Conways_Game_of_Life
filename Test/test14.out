___X_____X
____X_____
___X______
__X_______
___X____XX
____X__X__
__________
________X_
___X______
__X_______
__________
__________
__X_______
__X_______
__________
__________
__________
__________
__________
__________
